lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
+      raise ArgumentError.new unless dice
+      score = 0
+
+      # Create hash of count of each die
+      counts = dice.inject(Hash.new(0)) {|hash, num| hash[num] += 1; hash}
+
+      # Calculate score
+      counts.each do |die, count|
+        trips, remainder = count / 3, count % 3
+
+        case die
```

Comment:
I like this way of tallying score, but it looks really close to a more general solution. If we gave a name to `die * 100` and noticed that the only outlier is `die = 1`, which behaves remarkedly like `die = 10` could you simply this calculation to not require a case statement?